### PR TITLE
bareos-client 21.1.3

### PR DESCRIPTION
**Pull request by cho-m**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/bareos-client.rb [hunk 1]
```diff
@@ -21,29 +21,30 @@ class BareosClient < Formula
...
-                            "-Dfd-password=XXX_REPLACE_WITH_CLIENT_PASSWORD_XXX",
-                            "-Dmon-fd-password=XXX_REPLACE_WITH_CLIENT_MONITOR_PASSWORD_XXX",
-                            "-Dbasename=XXX_REPLACE_WITH_LOCAL_HOSTNAME_XXX",
-                            "-Dhostname=XXX_REPLACE_WITH_LOCAL_HOSTNAME_XXX",
-                            "-Dclient-only=ON"
-      system "make"
-      system "make", "install"
-    end
+    args = %W[
+      -DENABLE_PYTHON=OFF
```
Comment (cho-m) [inline Formula/bareos-client.rb:28]: No idea how to only disable Python 2.

`bareos-client` uses `find_package(Python2 ...)` and `find_package(Python3 ...)`. - https://github.com/bareos/bareos/blob/Release/21.1.2/core/cmake/BareosFindAllLibraries.cmake#L45-L46

File: Formula/bareos-client.rb [hunk 2]
```diff
@@ -21,29 +21,40 @@ class BareosClient < Formula
...
-                            "-Dmon-fd-password=XXX_REPLACE_WITH_CLIENT_MONITOR_PASSWORD_XXX",
-                            "-Dbasename=XXX_REPLACE_WITH_LOCAL_HOSTNAME_XXX",
-                            "-Dhostname=XXX_REPLACE_WITH_LOCAL_HOSTNAME_XXX",
-                            "-Dclient-only=ON"
-      system "make"
-      system "make", "install"
-    end
+    # Work around hardcoded paths to /usr/local Homebrew installation
+    inreplace "core/CMakeLists.txt", "/usr/local/", "#{HOMEBREW_PREFIX}/"
+    inreplace "core/cmake/FindReadline.cmake", "/usr/local/opt/readline/lib", Formula["readline"].opt_lib
```
Comment (cho-m) [inline Formula/bareos-client.rb:35]: Hardcoded path to readline static library (`/usr/local/opt/readline/lib/libreadline.a`) in https://github.com/bareos/bareos/blob/Release/21.1.2/core/cmake/FindReadline.cmake#L32=

Could just remove the `Darwin` section and let CMake `find_library` properly detect the dylib.

File: Formula/bareos-client.rb (hunk 1 above)
Comment (SMillerDev) [reply Formula/bareos-client.rb:38]: Could we just `inreplace "Python2", "Python3"`?

File: Formula/bareos-client.rb (hunk 2 above)
Comment (SMillerDev) [reply Formula/bareos-client.rb:35]: I'd suggest removing the Darwin section indeed

File: Formula/bareos-client.rb (hunk 1 above)
Comment (cho-m) [reply Formula/bareos-client.rb]: Even manually disabling Python 2 causes other failures with Python 3 as it tries to install into Python formula Framework directory. For now, will drop Python plugins.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**